lib/fawry/contracts/charge_request_contract.rb

```diff
@@ -11,6 +11,7 @@ class ChargeRequestContract < Dry::Validation::Contract
         required(:amount).value(:decimal)
         required(:description).value(:string)
         required(:customer_mobile).value(:string)
+        required(:language).value(:string)
```

Comment:
Thank you for another great contribution! 🤝 did Fawry add this required param recently? it's in their integration documentation, but I don't remember seeing it before and it breaks backward compatibility 🤔

Reply:
it's my mistake :D, it's required on their docs, i saw it first time last week, but if we didn't send it it works well

<img width="897" alt="image" src="https://user-images.githubusercontent.com/10594965/129480434-63416a48-d0a0-444b-b7dc-181d33bd074b.png">


I'll update the pr later today to make it optional and add a default value

Reply:
@amrrbakry updated!

Reply:
no problem. their documentation is shit 😂 yes, I think that would be a good idea to make it optional and also make sure the value is either `"ar-eg"` or `"en-gb"` to match the documentation:

```ruby
optional(:language).value(included_in?: %w[ar-eg en-gb])
```
and we add a default value in the charge request default params here:

https://github.com/fawry-api/fawry/blob/0ed3abb9c1a4270e0f1e16d467008fc41524a1c7/lib/fawry/requests/charge_request.rb#L8

something like that 😄 that way we don't break backward compatibility too 👍 

Thank you so much 🤝 I will add you as a collaborator after merging this change, and I trust you will maintain this project well 💯